calculate_average_pdrakatos.sh

```diff
@@ -0,0 +1,4 @@
+source "$HOME/.sdkman/bin/sdkman-init.sh"
```

Comment:
Please name the file calculate_average_PanagiotisDrakatos.sh, as described in the README.